CHANGELOG.md

```diff
@@ -332,6 +332,23 @@
 - Update `CONTRIBUTION.md` about adding an empty line after the copyright header (#7706) (Thanks @iSazonov!)
 - Update docs about .NET Core version `2.0` to be about version `2.x` (#7467) (Thanks @bergmeister!)
 
+## v6.1.2 - 2019-01-15
+
+### Tests
+
+- Fix test failures (Internal 6310)
```

Comment:
I wonder - should it be `(Internal #6310)`?

Also maybe make real links for such references?
`[(Internal #6310](https://github.com/...)`